pkgs/applications/misc/todoist-electron/default.nix

```diff
@@ -1,12 +1,13 @@
-{ lib, stdenv, fetchurl, appimageTools, makeWrapper, electron_21, libsecret }:
+{ lib, stdenv, fetchurl, appimageTools, makeWrapper, electron_24, libsecret }:
 
 stdenv.mkDerivation rec {
   pname = "todoist-electron";
-  version = "1.0.9";
+  version = "8.3.2";
 
   src = fetchurl {
-    url = "https://electron-dl.todoist.com/linux/Todoist-${version}.AppImage";
-    sha256 = "sha256-DfNFDiGYTFGetVRlAjpV/cdWcGzRDEGZjR0Dc9aAtXc=";
+    # https://electron-dl.todoist.com/linux/Todoist-linux-x86_64-8.3.2.AppImage
```

Comment:
```suggestion
```
that comment is not helpful as in to discover where to download new versions or similar, we don't need to write out what the url would be with version substituted.